.github/workflows/smoketest-chart.yaml

```diff
@@ -0,0 +1,77 @@
+name: Helm Chart Smoketest
+
+on: [push, workflow_dispatch, pull_request]
+
+env:
+  K8S_VERSION: v1.32.3
+  MICROK8S_CHANNEL: 1.32/stable
+
+jobs:
+  helm-install-smoke-test:
+    permissions: {}
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        config:
+          - { type: "kind" }
+
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Install Helm
+        uses: Azure/setup-helm@v4
```

Comment:
Could we also pin this using a shasum?

Reply:
Sure thing!

Reply:
Btw. would be great to have dependabot version pinning and updating actions...